oam/etc/Columnstore.xml

```diff
@@ -189,6 +189,8 @@
 		<TotalUmMemory>25%</TotalUmMemory>
 		<CPUniqueLimit>100</CPUniqueLimit>
 		<AllowDiskBasedJoin>N</AllowDiskBasedJoin>
+		<ForceDiskBasedJoin>N</ForceDiskBasedJoin>
```

Comment:
I would suggest you to write a markdown file with these options and short explanations in doc directory. IMHO you shouldn't put these options into config. Nobody would use the first option except developers.

Reply:
Agree.. these options should be hidden from users and support

Reply:
UPD: After discussion, I'll put it into session variables and remove from config file.

Reply:
Much appreciated for the change